GliaWidgets/Sources/View/Chat/Message/GVAPersistentButtonView.swift

```diff
@@ -0,0 +1,101 @@
+import UIKit
+
+final class GvaPersistentButtonView: OperatorChatMessageView {
+    var onOptionTapped: ((GvaOption) -> Void)!
+
+    private let kLayoutMargins = UIEdgeInsets(top: 16, left: 16, bottom: 16, right: 16)
+    private let kImageHeight: CGFloat = 200.0
+
+    private let environment: Environment
+
+    init(with style: ChoiceCardStyle, environment: Environment) {
+        self.environment = environment
+        super.init(
+            with: style,
+            environment: .init(
+                data: environment.data,
+                uuid: environment.uuid,
+                gcd: environment.gcd,
+                imageViewCache: environment.imageViewCache,
+                uiScreen: environment.uiScreen
+            )
+        )
+    }
+
+    required init() {
+        fatalError("init() has not been implemented")
+    }
+
+    override func appendContent(_ content: ChatMessageContent, animated: Bool) {
+        switch content {
+        case let .gvaPersistentButton(choiceCard):
+            let contentView = self.contentView(for: choiceCard)
+            appendContentView(contentView, animated: animated)
+        default:
+            break
+        }
+    }
+
+    private func contentView(for persistentButton: GvaButton) -> UIView {
+        let containerView = UIView()
+        containerView.backgroundColor = UIColor(red: 0.953, green: 0.953, blue: 0.953, alpha: 1)
+        containerView.layer.cornerRadius = 8.49
+        containerView.layer.borderWidth = 2
+        containerView.layer.borderColor = UIColor.clear.cgColor
```

Comment:
Perhaps same with this color, although I don't know how is it handled for other border colors.

Reply:
fixed [4720b0e](https://github.com/salemove/ios-sdk-widgets/pull/683/commits/4720b0ee106d7a116f5fdceef08605e241372e8f)